package.json

```diff
@@ -102,6 +102,7 @@
     "sortablejs": "^1.4.2",
     "source-sans-pro": "^2.0.10",
     "spin.js": "^2.3.2",
+    "string_score": "^0.1.22",
```

Comment:
We have the soon to be removed Bloodhound (typeahead underlaying suggester) and Sifter.js (which is used in admin).
In your case I'm not sure they are relevant for embedded, so it's up to you wether you reuse Sifter or keep string_score